chipflow_lib/platforms/silicon.py

```diff
@@ -487,6 +488,7 @@ def _prepare(self, elaboratable, name="top"):
         return fragment.prepare(ports)
 
     def build(self, elaboratable, name="top"):
+        warnings.simplefilter(action="ignore", category=UnusedElaboratable)
```

Comment:
this seems like it might suppress useful warnings sometimes?

Reply:
I'm not sure - if you get any build failure at all you get pages of them, so not really useful at the moment.
Maybe could show them if build passes?

Reply:
yeah, showing them if it passes sounds good if that's not too hard to implement. otherwise I'm fine to go with this for now and just make a note for later to see if we can get this diagnostic back with less noise somehow.